src/AppBundle/Controller/InterviewController.php

```diff
@@ -306,4 +311,145 @@ public function bulkAssignAction(Request $request)
                 )),
         ));
     }
+
+    /**
+     * @param string $responseCode
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    private function userCancelFinal(string $responseCode, Interview $interview)
+    {
+        if ($interview === null) {
+            throw $this->createNotFoundException();
+        }
+
+        $interview->cancel();
+        $manager = $this->getDoctrine()->getManager();
+        $manager->persist($interview);
+        $manager->flush();
+
+        $this->notifyInterviewer(InterviewStatusType::CANCELLED, $interview);
+
+        $this->addFlash('success', 'Intervjuet ble kansellert.');
+
+        return $this->redirectToRoute('home');
+    }
+
+    /**
+     * @param string $responseCode
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function acceptByResponseCodeAction(string $responseCode)
+    {
+        $interview = $this->getDoctrine()->getRepository('AppBundle:Interview')->findByResponseCode($responseCode);
+
+        if ($interview === null) {
+            throw $this->createNotFoundException();
+        }
+
+        $interview->acceptInterview();
+        $manager = $this->getDoctrine()->getManager();
+        $manager->persist($interview);
+        $manager->flush();
+
+        $this->addFlash('success', 'Intervjuet ble akseptert.');
+
+        return $this->redirectToRoute('home');
+    }
+
+    /**
+     * @param string $responseCode
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function requestNewTimeAction(string $responseCode)
+    {
+        $interview = $this->getDoctrine()->getRepository('AppBundle:Interview')->findByResponseCode($responseCode);
+
+        if ($interview === null) {
+            throw $this->createNotFoundException();
+        }
+
+        $interview->requestNewTime();
+
+        $manager = $this->getDoctrine()->getManager();
+        $manager->persist($interview);
+        $manager->flush();
+
+        $this->notifyInterviewer(InterviewStatusType::REQUEST_NEW_TIME, $interview);
```

Comment:
Her kan du like greit utføre direkte: 
`$this->get('app.interview.manager')->sendRescheduleEmail($interview);`